sympy/stats/frv_types.py

```diff
@@ -136,8 +136,8 @@ def check(sides):
     @property
     @cacheit
     def dict(self):
-        as_int(self.sides) # Check that self.sides can be converted to an integer
-        return super(DieDistribution, self).dict
+        as_int(self.sides)
```

Comment:
I think that the comment makes sense. This is a typical thing that static code checking will warn about (output is not used), so the comment is useful to avoid removing it at some later stage.

Reply:
Oh! sorry, I removed it by mistake. Thanks for notifying.